README.md

```diff
@@ -26,7 +26,7 @@ const fastify = require('fastify')()
 fastify.register(require('@fastify/sensible'))
 
 fastify.get('/', (req, reply) => {
-  reply.notFound()
+  return reply.notFound()
```

Comment:
This is not needed. It becomes needed only when used inside an async function.

Reply:
Updated, and added a separate example for returning the method in an async handler.